network-api/networkapi/wagtailpages/factory/blog.py

```diff
@@ -40,6 +40,10 @@ def add_category(post):
     post.category.add(choice(categories))
     post.save()
 
+    internet_health = BlogPageCategory.objects.get(name='Internet Health')
```

Comment:
@Pomax would this be the correct way to rename IH in factory? Also, I updated the category name in the cms and looks like it may have fixed the [previous issue](https://github.com/mozilla/foundation.mozilla.org/pull/3849#pullrequestreview-312981442).